src/sap.ui.commons/src/sap/ui/commons/themes/base/globalErrorHandler.less

```diff
@@ -0,0 +1,24 @@
+.sapMMsgBoxGEH .expandableTextArea {
+	background-color: white;
+	height: 200px;
+	padding: 10px;
+}
+.sapMMsgBoxGEH .expandableTextArea textarea {
```

Comment:
For performance reasons, please try to avoid selectors that end with a plain HTML tag name and have no "direct child" operator before that tag name. There will not be too many "textarea" tags in the page, so this instance is not really an issue, and only a star ("*") in the end would be really bad, but it's good practice to use a ">" between parts of a selector if possible.
Not sure... is this a direct child in the end? If the structure is more complex, you might just ignore this comment or add a class name to the HTML tag in critical cases.